src/in_toto.rs

```diff
@@ -91,14 +91,15 @@ pub mod envelope {
         .context("creating SPDX envelope")
     }
 
-    pub fn scai<A>(source: &SourceCode, attestation: A) -> Result<Envelope>
+    pub fn scai<A, S>(source: &SourceCode, spdx: S, attestation: A) -> Result<Envelope>
     where
         A: AsRef<[u8]>,
+        S: AsRef<[u8]>,
     {
         Envelope::new(
             serde_json::json!({
                 "_type": SCHEMA_STATEMENT,
-                "subject": [ resource_descriptor(source.name, &source.tarball) ],
+                "subject": [ resource_descriptor(format!("{}.spdx.json", source.name), &spdx) ],
```

Comment:
The name makes sense but it would be better to compute the digest (sha256) of the generated SPDX. Either of the whole envelope of just the SPDX itself.

Reply:
It currently uses the digest of the generated SPDX; not the whole envelope.